finder-app/Makefile

```diff
@@ -0,0 +1,14 @@
+Compiler := gcc
+
+ifeq ($(CROSS_COMPILE),aarch64-none-linux-gnu-)
```

Comment:
@JayashCUB see assignment slides/video for some simplified logic you can use here.